admin-client/src/pages/Users.svelte

```diff
@@ -1,49 +1,28 @@
 <script>
   import { fetchUsers } from '../lib/api';
-  import { router } from '../stores';
   import {
-    Await,
-    GridContainer,
-    PageTitle,
-    PaginationBanner,
+    PaginatedQueryPage,
     UserTable,
   } from '../components';
-  
-  const limits = [
-    '10', '25', '50', '100',
-  ];
 
-  const defaultParams = {
-    limit: '25',
-    page: 1,
+  const fields = {
```

Comment:
Like the fields filter and it's portability between different models. We could add `role` for users here.

Reply:
wellll, a `role`  is only defined for a user in the context of an organization, so it would require organization to be applied, might be more than I want to bite off right now :)

Reply:
👍